README.md

```diff
@@ -129,7 +135,7 @@ Initialize Snap JS when customer click pay button
   </body>
 ```
 
-### Snap Redirect
+###Snap redirect
```

Comment:
You need to add spacing between `###` and the word. Look at this, it fail to render as proper heading: https://github.com/veritrans/veritrans-ruby/blob/441149af8176155e1ce667de1768b9c3cb93c1eb/README.md#2-choose-productmethod